### PR TITLE
Fix for #745.

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by tisto** [comment]
http://jenkins.plone.org/job/pull-request-5.0/116/

**Comment by tisto** [comment]
Build is green. If somebody can review and merge it, @esteele can make a release and include it in Plone 5.0b3.

File: plone/app/upgrade/v50/betas.py [hunk 1]
```diff
@@ -206,8 +206,58 @@ def to50beta2(context):
...
+
 
 def to50beta3(context):
     """5.0beta2 -> 5.0beta3"""
     loadMigrationProfile(context, 'profile-plone.app.upgrade.v50:to50beta3')
     loadMigrationProfile(context, 'profile-plone.app.querystring:upgrade_to_8')
+    portal = getSite()
+    cp_tool = getToolByName(portal, 'portal_controlpanel')
+    group_ids = [x.get('id') for x in cp_tool.getGroups()]
+    for group_id in group_ids:
```
Comment (jaroel) [inline plone/app/upgrade/v50/betas.py]: What are group_ids and group_id for?

Comment (jensens) [inline plone/app/upgrade/v50/betas.py]: Looks like some leftover for me, executing the same code ``len(group_ids)`` times, in fact it does not not hurt but is confusing and superfluos.

**Comment by jensens** [comment]
http://jenkins.plone.org/view/All/job/pull-request-5.0/129/
http://jenkins.plone.org/view/All/job/pull-request-4.3/55/

**Comment by esteele** [comment]
Thanks. Released.

**Pull request merged**